src/GlobalStyles.js

```diff
@@ -0,0 +1,20 @@
+import { createGlobalStyle } from 'styled-components'
+
+const GlobalStyles = createGlobalStyle`
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: 'Roboto',sans-serif;
```

Comment:
Did you add `Roboto` via google fonts in the index.html? Don't see that file here.